alerter/alerter.cs

```diff
@@ -1,43 +1,44 @@
 ﻿using System;
+using System.Diagnostics;
 
-namespace AlerterSpace
-{
-    class Alerter
-    {
+namespace TemperatureSpace {
+    class Alerter {
         static int alertFailureCount = 0;
-        static int codeOk = 200;
-        static int codeNotOk = 500;
-        static int tempThresholdInC = 200;
-        static int networkAlertStub(float celcius)
-        {
-            Console.WriteLine("ALERT: Temperature is {0} celcius, Threshold: {1} celcius",
-                celcius, tempThresholdInC);
-            // Return code 200 when successful in alerting breach on network
-            // Return code 500 for if failed in alerting breach on network
-            // stub always succeeds and returns 200
-            return codeOk;
-        }
-        static void alertInCelcius(float farenheit)
-        {
-            float celcius = (farenheit - 32) * 5 / 9;
-            int returnCode = networkAlertStub(celcius);
-            if (returnCode == codeNotOk)
+        static int tempThresholdInC = 280;

```

Comment:
Shall we remove the threshold business itself, and make it a pure temperature-transmitter?
Then alertInCelcius will need to take an instance of network-transmitter, or get an instance via a factory. We can make that instance to pass or fail.
For many, it would be the first time they are writing mocks. I feel let's leave out all other distractions...